core/src/main/java/org/elasticsearch/search/suggest/Suggesters.java

```diff
@@ -54,9 +54,9 @@ public Suggesters(Map<String, Suggester> suggesters, ScriptService scriptService
 
     private static Map<String, Suggester> addBuildIns(Map<String, Suggester> suggesters, ScriptService scriptService, IndicesService indexServices) {
```

Comment:
We can clean up `ScriptService` and `IndicesService` from the params and also the Suggester ctor?